cmd/restore.go

```diff
@@ -146,6 +147,8 @@ func (x *Restore) Execute(args []string) error {
 		cfg.Bootstrap = testnetBootstrapAddrs
 		dhtopts.ProtocolDHT = "/openbazaar/kad/testnet/1.0.0"
 		bitswap.ProtocolBitswap = "/openbazaar/bitswap/testnet/1.1.0"
+	} else {
+		bitswap.ProtocolBitswap = "/openbazaar/bitswap/1.1.0"
```

Comment:
How did you know to change/add this? Are there tests around Restore?

Reply:
This was set in our fork of go-ipfs before. Since we're no longer using a fork it needs to be set in ob-go. Both here and in start.